lib/std/zig/AstGen.zig

```diff
@@ -4298,12 +4298,12 @@ fn fnDeclInner(
                             &[_]u32{
                                 try astgen.errNoteNode(
                                     type_expr,
-                                    "if this is a name, annotate its type '{s}: T'",
+                                    "if this is a name, annotate its type: '{s}: Type'",
```

Comment:
Hm, not sure I agree with this one. `T` is clearly not a keyword or anything, whereas `Type` could be misconstrued as a specific syntax. I think `T` is better.